stories/src/staticLegend.stories.tsx

```diff
@@ -0,0 +1,249 @@
+import * as React from 'react'
+import {storiesOf} from '@storybook/react'
+import {withKnobs, number, select, boolean, text} from '@storybook/addon-knobs'
+
+import {Config, Plot, timeFormatter} from '../../giraffe/src'
+import {getRandomTable} from './data/randomTable'
+
+import {
+  PlotContainer,
+  xKnob,
+  yKnob,
+  xScaleKnob,
+  yScaleKnob,
+  fillKnob,
+  findStringColumns,
+  colorSchemeKnob,
+  legendFontKnob,
+  tickFontKnob,
+  showAxesKnob,
+  interpolationKnob,
+  timeZoneKnob,
+  tooltipOrientationThresholdKnob,
+  tooltipColorizeRowsKnob,
+} from './helpers'
+
+const maxValue = Math.random() * Math.floor(200)
+
```

Comment:
add documentation; of a minimalist example; ie:

 the Plot needs a 'staticLegend' config object to show the static legend;  this is the minimum needed:

```
 const staticLegend =  {
    heightRatio: .2,
        border: '1px solid orange',
        fontBrightColor: 'yellow',
  }
```

to be added to the plot's config object

Reply:
I applied to 0.2 ratio as a default rather than comment it in the storybook, which doesn't always interest everyone.

Agree that 0 as a default height makes no sense. Once a user has a `staticLegend` property, they should get a reasonable height as the default.

Reply:
hmm...if there are defaults for everything, if the user accepts all the defaults, then what does the staticLegend object look like?

an empty object?

or if they accept all defaults;
then something like:

```
const staticLegend = {show: true}
```
so something is in there.......
thoughts?